.goreleaser.yaml

```diff
@@ -0,0 +1,52 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+
+# The lines below are called `modelines`. See `:help modeline`
+# Feel free to remove those if you don't want/need to use them.
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
+# vim: set ts=2 sw=2 tw=0 fo=cnqoj
+
+version: 2
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod tidy
+    # you may remove this if you don't need go generate
+    - go generate ./...
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+    goarch:
+      - amd64
+      - arm64
+
+kos:
+  - repository: ghcr.io/barakplasma/whatsmeow-transcribe
```

Comment:
I have no idea what this does. I guess it is something docker-related. Do I need to change this URI to `ghcr.io/hoehermann/whatsmeow-transcribe` when merging?

Reply:
Yes, you have to change the URL so that it pushes the kos generated docker image to your GitHub packages. I haven't tested the images yet, so I don't recommend merging this just yet